cmd/operator-sdk/bundle/create.go

```diff
@@ -46,65 +46,94 @@ func newCreateCmd() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "create",
 		Short: "Create an operator bundle image",
-		Long: `The 'operator-sdk bundle create' command will build an operator
-bundle image containing operator metadata and manifests, tagged with the
-provided image tag.
-
-To write all files required to build a bundle image without building the
-image, set '--generate-only=true'. A bundle Dockerfile, bundle metadata, and
-a 'manifests/' directory containing your bundle manifests will be written if
-'--generate-only=true':
-
-	$ operator-sdk bundle create --generate-only --directory ./deploy/olm-catalog/test-operator/0.1.0
-	$ ls .
-	...
-	bundle.Dockerfile
-	...
-	$ tree ./deploy/olm-catalog/test-operator/
-	└── 0.1.0
-		└── example.com_tests_crd.yaml
-		└── test-operator.v0.1.0.clusterserviceversion.yaml
-	└── manifests
-		└── example.com_tests_crd.yaml
-		└── test-operator.v0.1.0.clusterserviceversion.yaml
-	└── metadata
-		└── annotations.yaml
-
-'--generate-only' is useful if you want to build an operator's bundle image
-manually, modify metadata before building an image, or want to generate a
-'manifests/' directory containing your operator manifests for compatibility
+		Long: `
+The 'operator-sdk bundle create' command will build an operator bundle image containing operator metadata and 
+manifests, tagged with the provided image tag. 
+		
+To write all files required to build a bundle image without building the image, set '--generate-only=true'. 
+		
+Notes:
```

Comment:
I don't like the disorganized "list of things you should know" format in general. These bits of information should be woven into paragraphs that explain relevant behavior. Furthermore a bunch of these points are not specific to `bundle create`, and are explained more thoroughly by operator-registry docs which are already linked.